src/python/grpcio_tests/tests_aio/unit/metadata_test.py

```diff
@@ -212,8 +208,8 @@ async def test_trailing_metadata(self):
 
     async def test_from_client_to_server_with_list(self):
         multicallable = self._client.unary_unary(_TEST_CLIENT_TO_SERVER)
-        call = multicallable(
-            _REQUEST, metadata=list(_INITIAL_METADATA_FROM_CLIENT_TO_SERVER))
+        call = multicallable(_REQUEST,
+                             metadata=_INITIAL_METADATA_FROM_CLIENT_TO_SERVER)
```

Comment:
This test is explicitly guarding that we accept lists. Unfortunately, one of our big users depends on this behavior due to previous unclear definition of "sequence".